src/lib/modules/blockchain_process/blockchain.js

```diff
@@ -402,15 +406,15 @@ Blockchain.prototype.initChainAndGetAddress = function (callback) {
     function listAccounts(next) {
       self.runCommand(self.client.listAccountsCommand(), {}, (err, stdout, _stderr) => {
         if (err || stdout === undefined || stdout.indexOf("Fatal") >= 0) {
-          this.logger.info(__("no accounts found").green);
+          self.logger.info(__("no accounts found").green);
           return next();
         }
         let firstAccountFound = self.client.parseListAccountsCommandResultToAddress(stdout);
         if (firstAccountFound === undefined || firstAccountFound === "") {
           console.log(__("no accounts found").green);
           return next();
         }
-        this.logger.info(__("already initialized").green);
+        self.logger.info(__("already initialized").green);
```

Comment:
I'd **actually** prefer if we'd move from `self` to `this` and use fat arrow functions. I assume this has been changed for consistency reasons?

Reply:
I changed it because it crashed yes haha